eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FieldEmpty.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
```

Comment:
Can strict types be declared here? The same question applies to the previous file.